onelive/archive/index.html

```diff
@@ -33,20 +33,33 @@
 <body>
 
 <main>
-    <!--main content goes here-->
-    <section style="background: url('../images/header-onelive.svg') right bottom no-repeat; height: 70vh">
-        <!--Navbar is placed here to align the background image correctly-->
+    <section class="section">
         <div id="navbar"></div>
```

Comment:
Please add the comment on this navbar div @janithlahirukariyawasam

Reply:
@janithlahirukariyawasam can you do the above-mentioned change?

Reply:
Anjula I am currently outside from my home. So I didn't bring my laptop I will definitely do this work within Tomorrow.

Reply:
No worries! @janithlahirukariyawasam

Reply:
```suggestion
        <!--Navbar is placed here to align the background image correctly-->
        <div id="navbar"></div>
```

Reply:
@YoshithaRathnayake Can you check now? Sorry Guys I have just arrived at my house right now.

Reply:
> @YoshithaRathnayake Can you check now? Sorry Guys I have just arrived at my house right now.

Sure

Reply:
> 

Add the changes like this @janithlahirukariyawasam

Reply:
```suggestion
        <!--navbar goes here-->
        <div id="navbar"></div>
```